From 31f78092e977ec21e908209928750c6b755774ae Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Thu, 9 Aug 2018 15:40:07 +0300
Subject: [PATCH 0837/1200] config: mvebu: Add SD/eMMC definitions to Armada
 boards

Add definitions required for proper SD/eMMC support to common
Armada header file (partition number and device ID).

Change-Id: I0f013ae832bccb4dc731428d94701f959eb52eac
Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/58918
Reviewed-by: Igal Liberman <igall@marvell.com>
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
---
 include/configs/mvebu_armada-common.h | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/include/configs/mvebu_armada-common.h b/include/configs/mvebu_armada-common.h
index 1f0d229666..d6ac8731dd 100644
--- a/include/configs/mvebu_armada-common.h
+++ b/include/configs/mvebu_armada-common.h
@@ -124,12 +124,18 @@
  */
 #define CONFIG_ENV_SIZE			(64 << 10) /* 64KiB */
 #define CONFIG_ENV_SECT_SIZE		(64 << 10) /* 64KiB sectors */
+
 #ifdef CONFIG_MVEBU_NAND_BOOT
 #define CONFIG_ENV_OFFSET		0x400000
 #else
 #define CONFIG_ENV_OFFSET		(0x400000 - CONFIG_ENV_SIZE)
 #endif
 
+#ifdef CONFIG_ENV_IS_IN_MMC
+#define CONFIG_SYS_MMC_ENV_DEV		0
+#define CONFIG_SYS_MMC_ENV_PART		0 /* 0-DATA, 1-BOOT0, 2-BOOT1 */
+#endif
+
 /*
  * SATA/SCSI/AHCI configuration
  */
-- 
2.22.0

